pkg/package_manager_test.go

```diff
@@ -0,0 +1,51 @@
+package pkg
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestPackageManager_interpretFail(t *testing.T) {
```

Comment:
I don't see a value of this test.. what's your motivation?

Reply:
It's the best I can do without killing my local setup for testing 😄

Reply:
this test is testing string operation; it's too obvious, sry